### ISSUE TITLE
Invalid preamble for SM300D2 response

### ISSUE THREAD

**Issue by stephenjamieson** [moritzgloeckl/esp-air#1]
Encountering an issue after about an hour of successful reads.

```
[22:03:56][W][sm300d2:024]: Invalid preamble for SM300D2 response!
```

Thoughts?

**Comment by moritzgloeckl** [comment]
Is your sensor correctly wired to your ESP? Seems like a BAUD-rate / faulty connection error...

**Comment by stephenjamieson** [comment]
Yep, may be a faulty connection but I get data sometimes.  Maybe it's just a faulty unit

**Comment by moritzgloeckl** [comment]
Are you able to post the full raw output? You can connect it to the ESP or a Raspi via UART and read the raw values. Would be interesting what it sends...

**Comment by Duker4444** [comment]
Sample of that debug output from the UART.
I was able to find the data pattern and aligned them here.

01:03:0E:01:93:00:00:00:02:00:00:00:00:08:BD:0C:EC:A4:EF
01:03:0E:01:93:00:00:00:02:00:00:00:00:08:BB:0C:E6:C4:E9
01:03:0E:01:90:00:00:00:00:00:00:00:00:08:C0:0C:E5:E8:52
01:03:0E:01:90:00:00:00:00:00:00:00:00:08:BE:0C:E8:49:8F
01:03:0E:01:91:00:00:00:01:00:00:00:00:08:BB:0C:E6:D7:5B
01:03:0E:01:94:00:00:00:03:00:00:00:00:08:BD:0C:E6:22:3F
01:03:0E:01:93:00:00:00:02:00:00:00:00:08:BE:0C:E9:94:EC
01:03:0E:01:94:00:00:00:03:00:00:00:00:08:BD:0C:E8:A3:FB
01:03:0E:01:93:00:00:00:02:00:00:00:00:08:BB:0C:E6:C4:E9
01:03:0E:01:90:00:00:00:00:00:00:00:00:08:BD:0C:E1:79:89
01:03:0E:01:91:00:00:00:01:00:00:00:00:08:BB:0C:E5:97:5A
01:03:0E:01:91:00:00:00:01:00:00:00:00:08:BD:0C:E2:36:99
01:03:0E:01:90:00:00:00:00:00:00:00:00:08:BD:0C:E4:B9:8A
01:03:0E:01:91:00:00:00:01:00:00:00:00:08:BD:0C:E5:77:5B

These are two brand new sensors , It looks like they have started using 19 vs the old 17 placements.
and thus throwing off the reading and processing of the feed.
